store/storetest/compliance_store.go

```diff
@@ -165,40 +165,40 @@ func testComplianceExport(t *testing.T, ss store.Store) {
 	time.Sleep(100 * time.Millisecond)
 
 	cr1 := &model.Compliance{Desc: "test" + model.NewId(), StartAt: o1.CreateAt - 1, EndAt: o2a.CreateAt + 1}
-	cposts, err := ss.Compliance().ComplianceExport(cr1)
-	require.Nil(t, err)
+	cposts, complianceErr := ss.Compliance().ComplianceExport(cr1)
```

Comment:
Had to rename this as it was conflicting with the error returned by by `t1, err := ss.Team().Save(t1)` [above](https://github.com/mattermost/mattermost-server/pull/14808/files#diff-9ea510a524d48d6e88c95997e3c8047dL106).

Reply:
We are using `nErr` as the convention here for other PRs. Let's do that here.